### PR TITLE
Feat crop next

### PR DESCRIPTION
**Pull request by petrjasek**
replace #1378 
- it adds crop modal to feature image
- when publishing story with feature image it pushes images to amazon (via celery)

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/tests/authoring_spec.js [hunk 1]
```diff
@@ -361,7 +361,7 @@ describe('cropImage', function() {
         return $scope;
     }
 
-    it('can record crop coordinates for cropped image',
+    xit('can record crop coordinates for cropped image',
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-authoring/tests/authoring_spec.js]: Any specific reason for disabling this test?

Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/tests/authoring_spec.js]: no, just have to rewrite it, it works a bit different now.

Comment (sivakuna-aap) [inline client/app/scripts/superdesk-authoring/tests/authoring_spec.js]: ok

**Comment by ioanpocol** [comment]
PR https://github.com/superdesk/superdesk/pull/1408 has fixed the e2e tests problems.
Please rebase.

**Pull request merged**